legend-engine-language-pure-store-relational/src/main/java/org/finos/legend/engine/language/pure/grammar/from/RelationalParseTreeWalker.java

```diff
@@ -1098,7 +1098,7 @@ private PropertyMapping visitPropertyMappingWithLocalMappingProperty(RelationalP
         multiplicity.setUpperBound("*".equals(toMultiplicityContext.getText()) ? null : Integer.parseInt(toMultiplicityContext.getText()));
         PropertyPointer propertyPointer = new PropertyPointer();
         propertyPointer.property = PureGrammarParserUtility.fromIdentifier(ctx.identifier());
-        propertyPointer._class = localMappingPropertyInfo.type;
+        propertyPointer._class = _class;
```

Comment:
Don't think setting it to _class is correct. The class being mapped will not have this property. The mapping class which gets created at the compile time will be the owner. Hence, leaving it as null makes more sense.

Reply:
@gs-ssh16 updated the changes. Please take a look